vllm/model_executor/models/llava.py

```diff
@@ -273,19 +224,12 @@ def forward(
         This way, the `positions` and `attn_metadata` are consistent
         with the `input_ids`.
 
-        This model has two modes of image inputs:
-        `PIXEL_VALUES` and `IMAGE_FEATURES`.
-
         Args:
             input_ids: Flattened (concatenated) input_ids corresponding to a
                 batch.
             pixel_values: The pixels in each input image.
                 Expects a batch with shape `[1, 3, 336, 336]`.
                 (Only applicable to `PIXEL_VALUES` mode)
-            image_features: The image features for each input image outputted by
-                the vision tower before passing to the multi-modal projector.
-                Expects a batch with shape `[1, 576, 1024]`.
-                (Only applicable to `IMAGE_FEATURES` mode)
 
         See also:
             Each input maps to huggingface implementation, as follows:
```

Comment:
We can remove the documentation about mapping to HuggingFace inputs. Ditto for LLaVA-NeXT docs.